### ISSUE TITLE
Unable to build json-glib

### ISSUE THREAD

**Issue by jslick** [kiss-community/community#815]
I get `meson.build:1:0: ERROR: Unknown options: "docs"`

@Mederim

**Comment by mar-foo** [comment]
```diff
 meson \
     --prefix=/usr \
-    -Ddocs=false \
+    -Dgtk_doc=disabled \
     -Dintrospection=disabled \
     . output
```
This fixes the issue for me.

**Comment by Mederim** [comment]
Already send a fix

**Comment by dilyn-corner** [comment]
Closed by 6226444fb8c128e755f1f249396d1a4cfad4a485
